.github/workflows/_test_pax.yaml

```diff
@@ -41,11 +45,20 @@ jobs:
         id: meta
         shell: bash -x -e {0}
         run: |
-          TEST_CASE_NAME=1P${{ matrix.N_GPU }}G${{ matrix.DP }}DP${{ matrix.TP }}TP${{ matrix.PP }}PP
+          TEST_CASE_NAME=${{ matrix.PARALLEL_CONFIG[0] }}DP${{ matrix.PARALLEL_CONFIG[1] }}TP${{ matrix.PARALLEL_CONFIG[2] }}PP
+          TOTAL_TASKS=$((${{ matrix.PARALLEL_CONFIG[0] }} * ${{ matrix.PARALLEL_CONFIG[1] }} * ${{ matrix.PARALLEL_CONFIG[2] }}))
+          GPUS_PER_NODE=$TOTAL_TASKS
```

Comment:
How about consolidating lines 50-56 into
```
MAX_GPUS_PER_NODE=8
NODES=$((TOTAL_TASKS / MAX_GPUS_PER_NODE))
GPUS_PER_NODE=$((TOTAL_TASKS / NODES))
```
?

Reply:
I don't think this will quite do what we want because `NODES` needs to be the ceiling of `TOTAL_TASKS / MAX_GPUS_PER_NODE`. But I think that `NODES=$((result=(TOTAL_TASKS+MAX_GPUS_PER_NODE-1)/MAX_GPUS_PER_NODE))` should suffice.

Reply:
Good catch 👍